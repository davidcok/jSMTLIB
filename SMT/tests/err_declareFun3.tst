; declare-fun before set-logic
(declare-fun x () Bool)
