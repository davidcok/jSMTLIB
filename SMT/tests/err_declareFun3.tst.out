(declare-fun x () Bool)
^^^^^^^^^^^^^^^^^^^^^^^
(error "The logic must be set before a declare-fun command is issued")
